helm/hpcc/templates/_helpers.tpl

```diff
@@ -357,7 +331,7 @@ Add data volume
 Pass in dict with root
 */}}
 {{- define "hpcc.addDataVolume" -}}
-{{- include "hpcc.addVolumes" (dict "root" .root "includeLabels" (list "data" "" "lz") ) -}}
+{{- include "hpcc.addVolumes" (dict "root" .root "includeCategories" (list "data" "" "lz") ) -}}
```

Comment:
since we're breaking backward compatibility anyway, can't we remove checking for "" ?

Reply:
I think "" left in by mistake here? since category is now required

Reply:
yes, I'll remove